### ISSUE TITLE
objective parameter of XGBoost Classifier

### ISSUE THREAD

**Issue by vudzung06011992** [dmlc/xgboost#9152]
Hi, I am using XGBoostClassifier for Binary Classification. From the page: https://xgboost.readthedocs.io/en/latest/parameter.html, regarding the “objective” parameter, there are several options. I am confusing of their meaning:

reg:logistic: logistic regression
binary:logistic: logistic regression for binary classification, output probability
From the above description, I understand that if I set objective = “reg:logistic” or objective = “binary:logistic”; the model turns out to be logistic regression (kind of simple and linear model; not Gradient boosting any more). Is that true? If not, please let me know why the name “logistic regression” is used for describing reg:logistic and binary:logistic.

After all, what are possible options for XGBoost Classifier for binary classification? I want output to be score (kinds of probability, not 0/1 prediction).

Besides, what are the difference between XGBoost from https://xgboost.readthedocs.io/en/latest/parameter.html and sklearn.

I am looking forward to hearing from you.
Thank you.

**Comment by vudzung06011992** [comment]
If using "binary:logistic" objective, does XGBoost become Logistic Regression model?

**Comment by trivialfis** [comment]
Yes, but to emphasize the correction for this comment:

> the model turns out to be logistic regression (kind of simple and linear model; not Gradient boosting any more). 

When you use `binary:logistic` or `reg:logistic`, XGBoost builds **gradient boosting** logistic model.
